tests/test_ydocs.py

```diff
@@ -53,3 +55,14 @@ def test_ynotebook_undo_manager():
     ynotebook.undo_manager.undo()
     assert len(ynotebook.ycells) == 0
     assert not ynotebook.undo_manager.can_undo()
+
+
+def test_awareness():
+    yblob = YBlob()
+
+    assert yblob.awareness is None
+
+    awareness = Awareness(yblob.ydoc)
+    yblob.awareness = awareness
+
+    assert yblob.awareness == awareness
```

Comment:
A more usefull test would be:
```suggestion
    yblob = YBlob()
    assert yblob.awareness is None

    ydoc = Doc()
    awareness = Awareness(ydoc)
    yblob = YBlob(ydoc, awareness)
    assert yblob.awareness == awareness
```